src/Altinn.Apps/AppTemplates/AspNet/App.IntegrationTests/Utils/SetupUtil.cs

```diff
@@ -120,7 +120,7 @@ public static class SetupUtil
                             services.AddSingleton<IAltinnApp, IntegrationTests.Mocks.Apps.Ttd.EFormidlingInvalid.App>();
                             break;
                         case "presentationfields-app":
-                            services.AddSingleton<IAltinnApp, IntegrationTests.Mocks.Apps.Ttd.PresentationTextsApp.App>();
+                            services.AddScoped<IAltinnApp, IntegrationTests.Mocks.Apps.Ttd.PresentationTextsApp.App>();
```

Comment:
would it be a point to change all the apps to scoped to ensure that we don't break any other existing logic or functionality used in the test apps? I don't think this is the case, but might be good to have the testdata reflect this?

Reply:
agree

Reply:
In Startup `IAltinnApp` is registered as Transient. We should do that here as well.

Reply:
done